packages/lab/src/carousel/CarouselSlide.tsx

```diff
@@ -1,76 +1,134 @@
-import { makePrefixer } from "@salt-ds/core";
+import { makePrefixer, useForkRef } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
 import {
-  type ElementType,
   type HTMLAttributes,
-  type ReactElement,
+  type ReactNode,
   forwardRef,
+  useEffect,
   useRef,
+  useState,
 } from "react";
-
-import { clsx } from "clsx";
+import { useCarousel } from "./CarouselContext";
+import carouselSlideCss from "./CarouselSlide.css";
 
 export interface CarouselSlideProps extends HTMLAttributes<HTMLDivElement> {
-  ButtonBar?: ElementType;
-  Media: ReactElement;
-  description?: string;
-  title?: string;
-  contentAlignment?: "center" | "left" | "right";
+  /**
+   * Actions to be displayed in the content footer.
+   */
+  actions?: ReactNode;
+  /**
+   * The media to be displayed inside the slide.
```

Comment:
description could be improved, it's not clear without reading the code, what media is and how it differs from children